src/modules/ergoCubEmotions/ergoCubEmotions.h

```diff
@@ -42,24 +43,57 @@ class Circle : public GraphicElement
 {
 public:
     cv::Point center;
-    int radius;
+    int radius {0};
 
     void draw(cv::Mat& img) override;
 
     static std::shared_ptr<GraphicElement> parse(const yarp::os::Bottle& options);
 };
 
-struct Stadium : public GraphicElement
+class Stadium : public GraphicElement

```

Comment:
Why did you need this change?
A struct is a class with all public members/functions

Reply:
True. I just wanted to uniform wrt the Circle and BaseClass that are classes